docs/static-pvc.md

```diff
@@ -215,6 +225,45 @@ spec:
   volumeMode: Filesystem
 ```
 
+If you are a [Rook](https://rook.io/) user, there are some differences, such as:
```

Comment:
The above example should be enough. Let's not have Rook specific example here and anyway there isn't any extra configuration needed for Rook.